ci/jobs/projects.yaml

```diff
@@ -11,6 +11,16 @@
      - master
 
 - project:
+    name: build-automation-promote
+    jobs:
+     - 'build-automation-promote-{release_config}'
+    release_config:
+      - 2.8-dev:
+         version: 2.8
+      - master:
+         version: 2.9
```

Comment:
The way versions are inserted here is a little silly. I wonder if we might want to just stash this value in the release config or otherwise derive it, somehow. I'm content to let it work this way for now and improve it later.